### ISSUE TITLE
Error when generating a feed

### ISSUE THREAD

**Issue by vitek-rostislav** [shopsys/shopsys#297]
### What is happening
When trying to generate feed, I got en error:
```
In QueryException.php line 116:
                                                                           
  [Doctrine\ORM\Query\QueryException]                                      
  Too few parameters: the query defines 5 parameters but you only bound 4  
                                                                           

Exception trace:
 Doctrine\ORM\Query\QueryException::tooFewParameters() at /var/www/html/vendor/doctrine/orm/lib/Doctrine/ORM/Query.php:317
 Doctrine\ORM\Query->_doExecute() at /var/www/html/vendor/doctrine/orm/lib/Doctrine/ORM/AbstractQuery.php:957
 Doctrine\ORM\AbstractQuery->executeIgnoreQueryCache() at /var/www/html/vendor/doctrine/orm/lib/Doctrine/ORM/AbstractQuery.php:912
 Doctrine\ORM\AbstractQuery->execute() at /var/www/html/packages/framework/src/Model/Feed/Standard/StandardFeedItemRepository.php:61
 Shopsys\FrameworkBundle\Model\Feed\Standard\StandardFeedItemRepository->getItems() at /var/www/html/packages/framework/src/Model/Feed/FeedFacade.php:153
 Shopsys\FrameworkBundle\Model\Feed\FeedFacade->generateFeedBatch() at /var/www/html/packages/framework/src/Model/Feed/FeedFacade.php:83
 Shopsys\FrameworkBundle\Model\Feed\FeedFacade->generateStandardFeedsIteratively() at /var/www/html/packages/framework/src/Model/Feed/FeedCronModule.php:48
 Shopsys\FrameworkBundle\Model\Feed\FeedCronModule->iterate() at /var/www/html/packages/framework/src/Component/Cron/CronModuleExecutor.php:49
 Shopsys\FrameworkBundle\Component\Cron\CronModuleExecutor->runModule() at /var/www/html/packages/framework/src/Component/Cron/CronFacade.php:97
 Shopsys\FrameworkBundle\Component\Cron\CronFacade->runModule() at /var/www/html/packages/framework/src/Component/Cron/CronFacade.php:83
 Shopsys\FrameworkBundle\Component\Cron\CronFacade->runModuleByServiceId() at /var/www/html/packages/framework/src/Command/CronCommand.php:106
 Shopsys\FrameworkBundle\Command\CronCommand->runCron() at /var/www/html/packages/framework/src/Command/CronCommand.php:67
 Shopsys\FrameworkBundle\Command\CronCommand->execute() at /var/www/html/vendor/symfony/symfony/src/Symfony/Component/Console/Command/Command.php:251
 Symfony\Component\Console\Command\Command->run() at /var/www/html/vendor/symfony/symfony/src/Symfony/Component/Console/Application.php:964
 Symfony\Component\Console\Application->doRunCommand() at /var/www/html/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Console/Application.php:86
 Symfony\Bundle\FrameworkBundle\Console\Application->doRunCommand() at /var/www/html/vendor/symfony/symfony/src/Symfony/Component/Console/Application.php:248
 Symfony\Component\Console\Application->doRun() at /var/www/html/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Console/Application.php:74
 Symfony\Bundle\FrameworkBundle\Console\Application->doRun() at /var/www/html/vendor/symfony/symfony/src/Symfony/Component/Console/Application.php:148
 Symfony\Component\Console\Application->run() at /var/www/html/project-base/app/Bootstrap.php:58
 Shopsys\Bootstrap->run() at /var/www/html/project-base/bin/console:17
```

### Expected result
All feeds can be generated.

**Comment by vitek-rostislav** [comment]
This will probably be solved in https://github.com/shopsys/shopsys/pull/271 (I hope so :slightly_smiling_face:)

**Comment by vitek-rostislav** [comment]
Resolved in release [v7.0.0-alpha3](https://github.com/shopsys/shopsys/releases/tag/v7.0.0-alpha3)
